testsuite/integration-arquillian/tests/base/src/test/resources/import/testrealm-authenticator-config-null.json

```diff
@@ -0,0 +1,564 @@
+{
+     "authenticationFlows": [
+        {
+           "alias": "browser",
+           "authenticationExecutions": [
+              {
+                 "authenticator": "auth-spnego",
+                 "authenticatorFlow": false,
+                 "priority": 20,
+                 "requirement": "ALTERNATIVE",
+                 "userSetupAllowed": false
+              }
+           ],
+           "builtIn": true,
+           "description": "browser based authentification2",
+           "id": "3e6ccf87-5473-4eb0-8cbb-28f6b9e6f4d6",
+           "providerId": "basic-flow",
+           "topLevel": true
+        }
+     ],
+     "clientScopes": [
```

Comment:
Minor: Is it needed that this JSON contains "clientScopes" and 'userFederationMappers" and "userFederationProviders" ? AFAIK not and in this case, I vote for removing the unneeded parts to make it easier for the maintenance in the future.

Reply:
removed